### ISSUE TITLE
undefined symbol: _Static_assert - outdated gcc?

### ISSUE THREAD

**Issue by annaswims** [lfittl/pg_query#73]
I'm getting the error  "undefined symbol: _Static_assert".

(just to prove that the db connection works)
````
$ bundle exec irb
irb(main):001:0> require 'pg'
=> true
irb(main):002:0> p = PGconn.open(:dbname => 'dbname', :port => 5432, :password=>"MYPASSWORD")
=> #<PG::Connection:0x00000001412110>
irb(main):003:0> statement = "SELECT * from people where id = 1"
=> "SELECT * from people where id = 1"
irb(main):004:0> res = p.exec(statement)
=> #<PG::Result:0x00000001ac40a8 @connection=#<PG::Connection:0x00000001412110>>
````
And the real problem...
````
irb(main):008:0> require 'pg_query'
=> true
irb(main):009:0> PgQuery.parse(statement)
irb: symbol lookup error: /applications/mobi/shared/bundle/ruby/extensions/x86_64-linux/pg_query-0.13.3/pg_query/pg_query.so: undefined symbol: _Static_assert
`````

Is the problem that I'm using gcc version 4.4.7?  My c knowledge is minimal, and I was hoping someone could confirm that this is the issue before I ask our sysadmin to consider updating gcc.

**Comment by lfittl** [comment]
Hi @annaswims - thanks for the report.

It seems right now libpg_query requires at least gcc 4.6.0, but we should be good to loosen that restriction a bit to allow earlier versions as well. Let me see if I can get a quick bugfix release out for that today.

**Comment by annaswims** [comment]
That's wonderful, but don't stress if it's too hard, we can just create a vm with a newer OS/gcc for our use.

**Comment by lfittl** [comment]
@annaswims A fix for this is now released in version `0.13.4` - if you have a chance would be great if you could verify that indeed fixes it :)

Thanks again for the report!

**Comment by annaswims** [comment]
That fixed it.  Thanks so much!!!
